src/components/RegistrationList.tsx

```diff
@@ -198,6 +204,7 @@ export const RegistrationListItemContent = ({
             <IconButton
               data-testid={`edit-registration-${identifier}`}
               component={Link}
+              target={target ?? ''}
```

Comment:
ville sagt at `undefined` (default) er bedre enn å alltid sette tom streng her, selv om jeg vil anta at alle nettlesere tolker det likt
```suggestion
              target={target}
```